core/app_modules/cal.js

```diff
@@ -21,6 +28,8 @@ module.exports.addCalendarEvent = co.wrap(function*(username, event, user){
   }
   if (!isOverlap) {
     yield client.nonQuery('INSERT INTO calendar("user", "time", room, duration, title)VALUES ($1, $2, $3, $4, $5);', [user, event.time, event.room, event.duration, event.title]);
+    user_info = yield get_user_info(user);
+    email.roomConfirmation(user_info, event.time, event.room)

```

Comment:
I think that this function isn't a request function. This is the add function. I strongly feel that the request function should be something different.